src/tasks/processIssueComment/probot-processIssueComment.js

```diff
@@ -149,7 +149,7 @@ async function probotProcessIssueComment({ context, commentReply, analytics }) {
         })
         const optionsConfig = await setupOptionsConfig({ repository })
 
-        repository.skipCi = optionsConfig.options.skipCi
+        repository.skipCi = Boolean(optionsConfig.options.skipCi) ? repository.skipCi : ''
```

Comment:
maybe rename the property for clarity, as the same name is used for a Boolean now? For example

```suggestion
        repository.skipCiString = Boolean(optionsConfig.options.skipCi) ? repository.skipCi : ''
```

Reply:
@gr2m Seems sensible.